dev-docs/source/dynamic_api.rst

```diff
@@ -1,11 +1,20 @@
 Dynamic API Endpoints
 =====================
 
+The project follows the standard Django REST Framework "router" logic
+together with viewsets.
+
+There is one twist.
 Since all :doc:`models are dynamically constructed <dynamic_models>`,
 the views, URL's and serializers are also dynamically constructed at startup.
```

Comment:
Views and serializers, i.e., the instances, are always constructed at runtime. What we do differently is that the view and serializer *classes* are also dynamically constructed.

Reply:
Good call! I've updated the intro even more. Should make it much easier to grasp the whole thing.